suites/squid/nvmeof/tier-2_nvmeof_1gwgroup_8gwnodes_ha_tests.yaml

```diff
@@ -0,0 +1,152 @@
+              service: osd
+              args:
+                all-available-devices: true
+      desc: RHCS cluster deployment using cephadm
+      destroy-cluster: false
+      module: test_cephadm.py
+      name: deploy cluster
+
+  - test:
+      abort-on-fail: true
+      config:
+        command: add
+        id: client.1
+        nodes:
+          - node14
+        install_packages:
+          - ceph-common
+        copy_admin_keyring: true
+      desc: Setup client on NVMEoF gateway
+      destroy-cluster: false
+      module: test_client.py
+      name: configure Ceph client for NVMe tests
+      polarion-id: CEPH-83573758
+
+# 8GW HA 4-subsystems multinode(7/8) Failover and failback parallely
+  - test:
+      abort-on-fail: true
+      config:
+        rbd_pool: rbd
+        gw_group: gw_group1
+        do_not_create_image: true
+        rep-pool-only: true
+        rep_pool_config:
+          pool: rbd
+        install: true                           # Run SPDK with all pre-requisites
+        cleanup:
+          - pool
+          - gateway
+          - initiators
+        gw_nodes:
+          - node6
+          - node7
+          - node8
+          - node9
+          - node10
+          - node11
+          - node12
+          - node13
+        subsystems:                             # Configure subsystems with all sub-entities
+          - nqn: nqn.2016-06.io.spdk:cnode1
+            serial: 1
+            bdevs:
+            - count: 2
+              size: 8G
+            listener_port: 4420
+            listeners: [node6, node7, node8, node9, node10, node11, node12, node13]
+            allow_host: "*"
+          - nqn: nqn.2016-06.io.spdk:cnode2
+            serial: 1
+            bdevs:
+            - count: 2
+              size: 8G
+            listener_port: 4420
+            listeners: [node6, node7, node8, node9, node10, node11, node12, node13]
+            allow_host: "*"
+          - nqn: nqn.2016-06.io.spdk:cnode3
+            serial: 1
+            bdevs:
+            - count: 2
+              size: 8G
+            listener_port: 4420
+            listeners: [node6, node7, node8, node9, node10, node11, node12, node13]
+            allow_host: "*"
+          - nqn: nqn.2016-06.io.spdk:cnode4
+            serial: 1
+            bdevs:
+            - count: 2
+              size: 8G
+            listener_port: 4420
+            listeners: [node6, node7, node8, node9, node10, node11, node12, node13]
+            allow_host: "*"
+        initiators:                             # Configure Initiators with all pre-req
+          - nqn: connect-all
+            listener_port: 4420
+            node: node14
+        fault-injection-methods:                # Failure induction
+          - tool: daemon
+            nodes:
+              - node6
+              - node7
+              - node8
+              - node9
+              - node10
+              - node11
+              - node12
+          - tool: daemon
```

Comment:
This should have been systemctl ?

Reply:
yes. I will include it long with another fail methods. For now, lets merge with 1 fail method i.e., daemon
I will update description